lib/src/server/interceptor.dart

```diff
@@ -11,4 +11,4 @@ import 'service.dart';
 /// If the interceptor throws [Exception], [GrpcError.internal] with exception.toString() will be returned.
 /// If the interceptor returns null, the corresponding [ServiceMethod] of [Service] will be called.
 typedef Interceptor = FutureOr<GrpcError> Function(
-    ServiceCall call, ServiceMethod method);
+    ServiceCall call, Service service, ServiceMethod method);
```

Comment:
What about just exposing this information through `ServiceCall`, e.g. you can add

```dart
class ServiceCall {
  /// ...
  Service get service;
  
  /// ...
  ServiceMethod get method;
}
```

Then you can access this information through `call` and you would not need to make a breaking change to `Interceptor` signature.